com.github.nrittsti.NTag.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.nrittsti.NTag
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: ntag
+finish-args:
```

Comment:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: iris
```

Looks like it could use `--device=dri` too.